xla/backends/gpu/codegen/cudnn_test.cc

```diff
@@ -895,7 +895,7 @@ ENTRY Test {
                             ErrorSpec{/*aabs=*/1e-3, /*arel=*/1e-5}));
 }
 
-TEST_F(CuDnnFusionExecutionTest, ConvWgradWithNHWCLayoutExecutesCorrectly) {
+TEST_F(CuDnnFusionExecutionTest, DISABLED_ConvWgradWithNHWCLayoutExecutesCorrectly) {
```

Comment:
Why do you disable the test completely? Shouldn't it be sufficient to skip it on B200?